src/Hint/List.hs

```diff
@@ -67,11 +69,14 @@ import Language.Haskell.GhclibParserEx.GHC.Types.Name.Reader
 
 
 listHint :: DeclHint
-listHint _ _ = listDecl
+listHint _ modu = listDecl overloadedListsOn
+  where
+    exts = nubOrd $ concatMap snd (languagePragmas (pragmas (ghcAnnotations modu)))
```

Comment:
Why the `nubOrd` here? It is moderately expensive, and I can't see any use for it

Reply:
Nice catch! It was originally used to pass down a clean `exts` to the following functions. Don't need it anymore for just `overloadedListsOn`, I have removed it in the latest commit.